packages/documentation/package.json

```
@@ -1,19 +1,22 @@
 {
-  "name": "docusaurus",
+  "name": "documentation",
```

Even though this package won't be published to npm, just for consistency with the other packages, I think this should be:

```suggestion
  "name": "@spaship/documentation",
```

---

Great idea! :+1: